src/main/kotlin/link/infra/packwiz/installer/ui/gui/ExceptionListWindow.kt

```diff
@@ -24,6 +24,18 @@ class ExceptionListWindow(eList: List<ExceptionDetails>, future: CompletableFutu
 		fun getExceptionAt(index: Int) = details[index].exception
 	}
 
+	private fun openUrl(url: String) {
+		try {
+			if (Desktop.isDesktopSupported() && Desktop.getDesktop().isSupported(Desktop.Action.BROWSE)) {
+				Desktop.getDesktop().browse(URI(url))
+			} else {
+				Runtime.getRuntime().exec(arrayOf("xdg-open", url));
```

Comment:
Could we test to see if `xdg-open` exists first? Then either grey out the buttons or just log an error in the console.

Reply:
I've found two ways to check. We could search `PATH` for the command, but that doesn't seem right to me or we could run `which xdg-open` and check the exit value.

We could check the exit value of running xdg-open and log an error if it failed somehow which I'd prefer due to it's simplicity.

Reply:
If the command is missing it throws an `IOException`, but I did add logging if the process exits with a non-zero code.